### ISSUE TITLE
Don't know what exactly causes it, but titles from the previous search appear in the new one

### ISSUE THREAD

**Issue by Dramorian** [zpix1/yt-anti-translate#10]
E.g, if I search for League of Legends Highlights and then go to She-Hulk Breakdown it shows League of Legends titles 
![image](https://user-images.githubusercontent.com/6598192/185700850-e570232b-aa41-4838-8826-5919779d6a53.png)
![image](https://user-images.githubusercontent.com/6598192/185700957-c5e88a52-1804-4c27-9fb5-25536dc25654.png)

Steps to recreate:
1) search for lol video
2) click on it
3) search for she-hulk

**Comment by Dramorian** [comment]
Also, can you make Tampermonkey compatible script?

**Comment by zpix1** [comment]
Pls verify it in 1.5.4

**Comment by Dramorian** [comment]
> Pls verify it in 1.5.4

seems like it doesn't appear anymore.

**Comment by Dramorian** [comment]
the original issue is gone, now titles disappear completely

**Comment by zpix1** [comment]
Yeah, YT constantly changes its page structure. Try again with 1.5.5

**Comment by zpix1** [comment]
Closing as obsolete. Please reopen if it still reproduces.
